assets/app/views/settings.html

```diff
@@ -12,7 +12,9 @@
         <dd ng-if="project | displayName: true">{{project | displayName: true}}</dd>
         <dd ng-if="!(project | displayName: true)"><em>No display name</em></dd>
         <dt>Description:</dt>
-        <dd ng-if="project | description">{{project | description}}</dd>
+        <dd ng-if="project | description">
+          <truncate-long-text content="project | description" limit="256"></truncate-long-text>
```

Comment:
could probably go higher than 256 here... not nearly as space-constrained as a tile on the create page